src/card-stack/card-stack.html

```diff
@@ -57,26 +93,56 @@
         return {
           stack: {
             type: Object
+          },
+          foundation:{
+            type: Boolean,
+            value: false
           }
         };
       }
 
+      /**
+      * Adds a card to the stack and generates a 'card-moved' event.
+      *
+      * @param {object} card - GameCard object
+      */
+      addCard(card){
+
+          this.set('stack.cards', this.stack.cards.concat([card]));
+          this.dispatchEvent(new CustomEvent('card-moved', {bubbles:true, composed:true}));
+      }
+
+      // Removes top card on stack.
+      popCard(){
+        this.set('stack.cards', this.stack.cards.slice(0, this.stack.cards.length -1));
```

Comment:
There's also `this.pop('stack.cards')` which might work. I'm not sure if I've used it but Polymer does provide several of these array mutation functions directly and takes care of automatically updating bindings.